gen/definitions/bgp_peer_address_family_prefix_list_control.yaml

```diff
@@ -0,0 +1,103 @@
+---
+name: BGP Peer Address Family Prefix List Control
+class_name: bgpPfxCtrlP
+dn: sys/bgp/inst/dom-[%s]/peer-[%s]/af-[%s]/pfxctrl-[%s]
+ds_description: This data source can read the BGP peer address family prefix list control configuration.
+res_description: This resource can manage the BGP peer address family prefix list control configuration.
+doc_path: Routing%20and%20Forwarding/bgp:PfxCtrlP/
+doc_category: BGP
+parents:
+  - BGP Peer Address Family
+attributes:
+  - nxos_name: asn
+    tf_name: asn
+    type: String
+    reference_only: true
+    description: 'Autonomous system number.'
+    example: '65001'
+  - nxos_name: vrf_name
+    tf_name: vrf
+    type: String
+    id: true
+    reference_only: true
+    description: 'VRF name.'
+    example: default
+  - nxos_name: addr
+    tf_name: address
+    type: String
+    id: true
+    reference_only: true
+    description: 'Peer address.'
+    example: '192.168.0.1'
+  - nxos_name: type
+    tf_name: address_family
+    type: String
+    id: true
+    reference_only: true
+    description: 'Address Family.'
+    enum_values:
+      - ipv4-ucast
+      - vpnv4-ucast
+      - ipv6-ucast
+      - vpnv6-ucast
+      - l2vpn-evpn
+      - lnkstate
+    default_value: 'ipv4-ucast'
+    example: 'ipv4-ucast'
+  - nxos_name: direction
+    tf_name: direction
+    type: String
+    id: true
+    description: 'Route Control direction.'
+    enum_values:
+      - in
+      - out
+    default_value: in
+    example: in
+  - nxos_name: list
+    tf_name: list
+    type: String
+    description: 'Route Control Prefix-List name.'
+    example: PREFIX_LIST1
+test_prerequisites:
+  - dn: sys/fm/bgp
+    class_name: fmBgp
+    no_delete: true
+    attributes:
+      - name: adminSt
+        value: enabled
+  - dn: sys/bgp
+    class_name: bgpEntity
+    dependencies: [0]
+  - dn: sys/bgp/inst
+    class_name: bgpInst
+    attributes:
+      - name: adminSt
+        value: enabled
+      - name: asn
+        value: '65001'
+    dependencies: [1]
+  - dn: sys/bgp/inst/dom-[default]
+    class_name: bgpDom
+    attributes:
+      - name: name
+        value: default
+    dependencies: [2]
+  - dn: sys/bgp/inst/dom-[default]/peer-[192.168.0.1]
+    class_name: bgpPeer
+    attributes:
+      - name: addr
+        value: '192.168.0.1'
+      - name: asn
+        value: '65001'
+    dependencies: [3]
+  - dn: sys/bgp/inst/dom-[default]/peer-[192.168.0.1]/af-[ipv4-ucast]
+    class_name: bgpPeerAf
+    attributes:
+      - name: addr
```

Comment:
We don't need to specific any parent keys here, as we already have the full dn.

Reply:
`addr` is not an attribute of `bgpPeerAf`, it is only used in the resource to identify the parent object. In this case we are pushing a single object with a full dn, so we only need to list the attributes of the actual object.